.eslintignore

```diff
@@ -7,5 +7,10 @@ src/**/__tests__/**
 # This should be enabled but that folder has too much in it that doesn't belong
 src/test
 test/the-files-to-test.generated.js
-# This is synced with a different file internally, don't want to lint it yet
-vendor/fbtransform/syntax.js
+vendor/
```

Comment:
Not sure about comment that was here before. Grunt task didn't lint `vendor/` before but comment was about `vendor/fbtransform/syntax.js` only.